### PR TITLE
Update Iroha SDK version and add tests to CI

### PR DESCRIPTION
**Pull request by ekovalev**
Iroha SDK version updated to ensure its compatibility with the latest Iroha binaries.
A sample testing scenario added to the CI to facilitate benchmarks.

## Checklist
 - [x]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [ ]  Documentation - any JSDoc, website, or Stackoverflow answers?

## Issue/User story

https://github.com/hyperledger/caliper/issues/725

## Steps to Reproduce

N/A

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-caliper)
- [ ] [GitHub Issues](https://github.com/hyperledger/caliper/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/caliper)

### CODE REVIEW

**Comment by nklincoln** [comment]
@ekovalev - thank you for this PR 👍 

Are you able to squash the commits?

File: packages/caliper-cli/lib/bind/config.yaml [hunk 1]
```diff
@@ -77,6 +77,8 @@ sut:
     iroha:
         0.6.3: &iroha-latest
             packages: [ 'iroha-helpers@0.6.3' ]
+        0.8.8: &iroha-latest
```
Comment (nklincoln) [inline packages/caliper-cli/lib/bind/config.yaml]: There should only be a single `&iroha-latest` pointer

**Comment by ryjones** [comment]
Notice this is on master not the iroha branch that you asked for

**Comment by nklincoln** [comment]
@ekovalev - apologies for the delay in looking at this one. For some reason the lint/license checker is failing on the build at the moment within the iroha package

**Pull request merged**